src/main/java/be/ugent/IfcSpfReader.java

```diff
@@ -252,6 +255,11 @@ public void setup(String ifcFileIn) throws IOException {
             }
 
             fis = IfcSpfReader.class.getResourceAsStream("/typ" + exp + ".ser");
+            
+			if (fis == null)
```

Comment:
Same here

Reply:
Hi,
Yes, it may be an Eclipse source folder issue (Exlipse:project:Properties:Java Build Path:Source resource filders). 
All the projects are created using Eclipse. The problem may be caused in the case my resource folder is empty in my project. In some cases, in GIT, empty folders are not always created, but it sould not affect the functionality of a library, should it? Anyhow, this bug has appeared many times. I'm currently using IFCtoRDF in seven projects: IFCtoLBD, IFCtoLBD_OpenAPI, IFCtoRDF-Desktop, ACLWoBD, Multi-AuthorityABE4BIM, ifcOWL-IFC-IPFS, and IFCtoB4R.

In my projects, I am using IFCtoRDF as a Maven library. When I have used Exclipse:export:Runnable Jar file to create an app, the developed app had not been able to find the resource files without the fix.  

To avoid extra lines in IFCypRDF, I will analyse ans test thsi further. I cannot take the risk of the bug reappearing when using IFCtoRDF from the Maven central.